gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -65,36 +65,30 @@ const NxCardPage = () =>
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card__header</code></td>
-            <td className="nx-cell">Element</td>
+            <td className="nx-cell"><code className="nx-code">.nx-card--horizontal</code></td>
+            <td className="nx-cell">Modifer of <code className="nx-code">.nx-card</code></td>
             <td className="nx-cell">
-              Card header element, typically wraps an <code className="nx-code">&lt;h3&gt;</code>.
+              Used for cards that are laid out horizontally rather than vertically.
```

Comment:
As it was originally, I think the horizontal/vertical nomenclature is very ambiguous.  Since these cards stack vertically they could just as easily (or perhaps even more easily) be thought of as "vertical cards"

Reply:
> As it was originally, I think the horizontal/vertical nomenclature is very ambiguous. Since these cards stack vertically they could just as easily (or perhaps even more easily) be thought of as "vertical cards"

I also thought of saying "wide rather than tall". I even considered removing the "wide" cards entirely since there is no demand for them at this point at all. I was trying to make a distinction in the language between how a card is oriented vs how cards are arrayed on the page.

Reply:
> I also thought of saying "wide rather than tall". I even considered removing the "wide" cards entirely since there is no demand for them at this point at all. I was trying to make a distinction in the language between how a card is oriented vs how cards are arrayed on the page.

I'm wondering if we can avoid the issue by simply getting rid of this class, and having all the styles that it provides attached to any `.nx-card` that is a child of a `.nx-card-container--column`

Reply:
> > I also thought of saying "wide rather than tall". I even considered removing the "wide" cards entirely since there is no demand for them at this point at all. I was trying to make a distinction in the language between how a card is oriented vs how cards are arrayed on the page.
> 
> I'm wondering if we can avoid the issue by simply getting rid of this class, and having all the styles that it provides attached to any `.nx-card` that is a child of a `.nx-card-container--column`

That's how it was originally. I moved away from it because I didn't like the container dictating the styling of the cards. I think that it's cleaner if the cards are responsible for their own internal layout and the container is responsible for the card's layout on the page, nothing more.

Initially it was based on my assumption that the ability to transform vertical cards to horizontal cards and back was a good idea. I think that was a bad assumption which made the cards overly complicated and too rigid in their layout.